docs/src/man/getting_started.md

```diff
@@ -23,6 +23,31 @@ Nullable()
 Nullable(1) + Nullable()
 ```
 
+The get() function is very usefull to extract the value from the Nullable wrapper.
+
+example :
+```julia
+julia> a = Nullable{String}("14:00:00")
+Nullable{String}("14:00:00")
+
+julia> typeof(a)
+Nullable{String}
+
+julia> b = Dates.DateTime(get(a), "HH:MM:SS")
```

Comment:
No need to introduce date/time functionalities here. Just show that the `String` value can be extracted.